src/components/Quiz/useQuizStore.test.ts

```
@@ -2846,4 +2875,327 @@
       expect(questionIndex).toBe(0);
     });
   });
+
+  describe('Question Result Functions', () => {
+    // Reset store state before each test in this describe block
+    beforeEach(() => {
+      act(() => {
+        useQuizStore.getState().setQuestionsResult(null as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)